### PR TITLE
Update configs

### PR DESCRIPTION
**Pull request by scott062**
DO NOT MERGE UNTIL EVERYTHING ELSE IS APPROVED AND MERGED

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
> DO NOT MERGE UNTIL EVERYTHING ELSE IS APPROVED AND MERGED

Should i merge it now?

File: public/config/config.json [hunk 1]
```diff
@@ -4,17 +4,11 @@
...
-    "bidding": true,
     "static_content": true,
-    "notifications": true,
-    "data_sync_admin": true,
-    "bid_count": true,
-    "personalization": true,
     "persona_auth": true,
     "client_counts": false,
     "cdo_season_filter": true,
     "cdo_bidding": true,
```
Comment (elizabeth-jimenez) [inline public/config/config.json:11]: cdo_bidding too

Comment (scott062) [reply public/config/config.json]: good catch

**Comment by SidelineCory24** [comment]
7 FF removed for this PR
image

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**